vm_supervisor/run.py

```diff
@@ -216,3 +214,30 @@ async def run_code_on_event(vm_hash: VmHash, event, pubsub: PubSub):
             execution.stop_after_timeout(timeout=settings.REUSE_TIMEOUT)
         else:
             await execution.stop()
+
+
+async def start_persistent_vm(vm_hash: VmHash, pubsub: PubSub) -> VmExecution:
+    execution: Optional[VmExecution] = await pool.get_running_vm(vm_hash=vm_hash)
+
+    if not execution:
+        logger.info(f"Starting persistent VM {vm_hash}")
+        execution = await create_vm_execution(vm_hash=vm_hash)
+    # If the VM was already running in lambda mode, it should not expire
+    # as long as it is also scheduled as long-running
+    execution.marked_as_persistent = True
```

Comment:
Maybe a function could be clearer, like execution.mark_as_persistent()?

Reply:
An extra function just to set an attribute with a similar name ? That looks like overhead-Java to me.

Reply:
Yeah I don't know, having a passive name like "marked as ..." begs for a function to me, style-wise. Otherwise I'd just name the attribute "persistent" or "is_persistent". But that's complete nitpicking, you can ignore this comment if you like it that way.

Reply:
Renamed